sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizePiiEntityAsyncClient.java

```diff
@@ -151,13 +151,13 @@
         final List<RecognizePiiEntitiesResult> recognizeEntitiesResults = new ArrayList<>();
         piiEntitiesResult.getDocuments().forEach(documentEntities -> {
             // Pii entities list
-            final List<PiiEntity> piiEntities = documentEntities.getEntities().stream().map(entity ->
-                new PiiEntity(entity.getText(), EntityCategory.fromString(entity.getCategory()),
+            final List<PiiEntity> piiEntities = documentEntities.getEntities().stream().map(
+                entity -> new PiiEntity(entity.getText(), EntityCategory.fromString(entity.getCategory()),
                     entity.getSubcategory(), entity.getConfidenceScore(), entity.getOffset()))
-                .collect(Collectors.toList());
+                                                    .collect(Collectors.toList());
```

Comment:
not sure if this is Java format or by error a bunch of spaces got introduced.

Reply:
it is my plugin tool's alignment. It aligned with `documentEntities`